bundles/org.eclipse.orion.client.ui/web/plugins/iframeCommunicationPlugin.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <script src="../orion/plugin.js"></script>
+  <script>
+  /*eslint-env browser, amd*/
+  /*global orion*/
+    window.onload = function() {
+      var provider = new orion.PluginProvider();
+      provider.registerService("orion.page.onload", {
```

Comment:
Do we need to register as a service, or can we just inline the registered code?  It looks like its just listening on window events.